resources/views/servicio/servicioPlanilla.blade.php

```diff
@@ -0,0 +1,260 @@
+@if($servicio)
+<table class="text-center table table-bordered">
+		<tr>
+			<td rowspan=2>B.V.</td>
+			<td class="text-center" colspan=14 rowspan=2>TRENQUE LAUQUEN</td>
+			<td colspan=2>INOBV</td>
+			<td colspan=2>RENI</td>
+			<td></td>
+		</tr>
+		<tr>
+			<td colspan=1>3</td>
+			<td colspan=1>3</td>
+			<td class="col-md-2" colspan=3>BA</td>
+		</tr>
+		<tr>
+			<td rowspan=2>PARTE DE SERVICIO</td>
+			<td>01</td>
+			<td>02</td>
+			<td>03</td>
+			<td>04</td>
+			<td>05</td>
+			<td>06</td>
+			<td>07</td>
+			<td>08</td>
+			<td>09</td>
+			<td>10</td>
+			<td>11</td>
+			<td>DIA</td>
+			<td>MES</td>
+			<td>AÑO</td>
+			<td colspan=2 rowspan=2>P.S.N°</td>
+			<td class="col-md-1" rowspan=2></td>
+			<td class="col-md-1" rowspan=2></td>
+			<td class="col-md-1" rowspan=2></td>
+		</tr>
+		<tr>
+			@if($servicio->tipo_servicio_id==1)
```

Comment:
Esto lo mismo, por ahi podes desde el controller crear un array con true/false de acuerdo al tipo de servicio y que con un for lo resolves.
@perezale Mañana por ahi le podes dar una mano a @NicoARodriguez  con esto.